nodejs/eks/examples/tests/awsx-network-and-subnetIds/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "test-awsx-network-and-subnetIds",
+    "devDependencies": {
+        "typescript": "^3.0.0",
+        "@types/node": "latest"
+    },
+    "dependencies": {
+        "@pulumi/pulumi": "^1.0.0-beta",
```

Comment:
Should be able to use 1.0.0 versions now

Reply:
Done